crates/walrus-sdk/src/utils.rs

```diff
@@ -3,16 +3,35 @@
 
 use std::{
     collections::HashMap,
-    fmt::{self, Display},
+    fmt::{self, Debug, Display},
+    future::Future,
     hash::Hash,
+    path::Path,
     time::Duration,
 };
 
-use anyhow::Result;
-use futures::{stream::FuturesUnordered, Future, StreamExt};
+use anyhow::{Context as _, Result};
+use futures::{stream::FuturesUnordered, StreamExt};
+use indicatif::{ProgressBar, ProgressStyle};
+use serde::de::DeserializeOwned;
 use tokio::time;
 use tracing::Level;
 
+/// Load the config from a YAML file located at the provided path.
+pub fn load_from_yaml<P: AsRef<Path>, T: DeserializeOwned>(path: P) -> anyhow::Result<T> {
```

Comment:
suggestion: add a TODO that this should probably go to walrus-utils crate

Reply:
Added a TODO. https://linear.app/mysten-labs/issue/WAL-764/move-load-from-yaml-into-walrus-utils